### ISSUE TITLE
cars seem to honk a lot

### ISSUE THREAD

**Issue by plexsoup** [Cataclysmanic/GO2022#147]
more than they used to?

Check into car states, make sure they stop honking after they crash or park
Also check their timers. increase the possible honk interval.

**Comment by plexsoup** [comment]
This is an ugly bug related to .gitignore, and filetype defaults.

I had originally imported the .ogg horn noises and then manually set them not to loop (because .ogg loops by default in godot). Fine. No problem.

Later, I checked out an older branch of the project. Then when I returned to the current branch, Godot reimported everything... with the default params. 

So, I had to go in and reset all the loop options on the .ogg files.

This will also be a problem if there are any .wav files we expected to loop, but aren't because the default for .wav is one-shot.

And probably any textures we set to tile.

Basically, any file where you have to "reimport" it with different values to get Godot to like it.

Ugh.. params on import is ugly.
